source/iface/stream.rst

```diff
@@ -119,3 +110,20 @@ compatibility.  get_max_statement_size() is a deprecated query.
    const sycl::stream_manipulator defaultfloat = sycl::stream_manipulator::defaultfloat;
    __precision_manipulator__ setprecision(int precision);
    __width_manipulator__ setw(int width);
+
+.. _stream-example:
+
+=========
+Example 1
```

Comment:
I am wondering whether to use: Example 1, 2, 3
or try to make the title descriptive. What do you think?